structuredheadings/plugin.js

```diff
@@ -65,6 +66,11 @@
         });
       };
 
+      //some friendly setup for level changes
+      var headerList = Object.keys(allowedElements);
```

Comment:
I've hardcoded the list for compatibility, it feels like writing the same thing twice (headerList vs allowedElements) but they are used in two different ways so... I don't know. Would be nice to have only one list.